### PR TITLE
Negar/careers

### PR DESCRIPTION
**Pull request by negar-binary**
add pdf of employee handbook to careers page for download and fix padding of footer for Indonesian language

### CODE REVIEW

File: src/sass/pages/careers.scss [hunk 1]
```diff
@@ -0,0 +1,25 @@
+.career {
+	.xquote {
+    	background-color: #F2F2F2;
+    	padding: 20px 20px;
+    	margin: 30px 0;
+		div {
```
Comment (raunak-binary) [inline src/sass/pages/careers.scss]: better to keep consistent tab space as sass depends on intendation

File: src/templates/haml/static/careers.html.haml [hunk 2]
```diff
@@ -23,10 +23,25 @@
...
+            %img.responsive{:src => url_for('images/pages/careers/corporate-culture.svg')}
+          %div.text
+            %a{:href => 'http://blog.wobb.my/project/binary/', :target => '_blank'}= l('Learn more about our corporate culture')
+    .grd-grid-6.grd-grid-mobile-12.grd-grid-phablet-6
+      %div.xquote
+        %div
+          %div.picture
+            %img.responsive{:src => url_for('images/pages/careers/handbook.svg')}
+          %div.text
+            %a{:href => url_for('/templates/haml/static/download/HB-employee-FA.pdf'), :download => true, :target => '_blank'}= ('Employee handbook')
```
Comment (raunak-binary) [inline src/templates/haml/static/careers.html.haml]: the localize tag is missing it should be `l('Employee handbook')`

File: src/sass/pages/careers.scss (hunk 1 above)
Comment (negar-binary) [inline src/sass/pages/careers.scss]: strange. in my local file the tabs are correctly spaced.

File: src/templates/haml/static/careers.html.haml (hunk 2 above)
Comment (negar-binary) [inline src/templates/haml/static/careers.html.haml]: ahh thanks :)

**Pull request merged**